types/events/pull_request_target/types.dhall

```diff
@@ -1 +1 @@
-< assigned | opened | synchronize | reopened >
+< assigned | opened | synchronize | reopened | closed >
```

Comment:
:art: This could be moved to a separate PR

Reply:
Sorry for the delay! https://github.com/regadas/github-actions-dhall/pull/69

Reply:
sorry for the delay!